benchmarks/src/jmh/java/com/linecorp/armeria/benchmarks/thrift/PooledResponseBufferBenchmark.java

```diff
@@ -5,7 +5,7 @@
  * version 2.0 (the "License"); you may not use this file except in compliance
  * with the License. You may obtain a copy of the License at:
  *
- *   https://www.apache.org/licenses/LICENSE-2.0
+ *   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Revert this line?

Reply:
Weird, my copyright style is old or wrong for some reason